example-wasm/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "@realm/example-wasm",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "start": "wireit",
+    "build": "wireit",
+    "eject": "react-scripts eject"
```

Comment:
Do we need this?  I tried running it for fun, and it produced an error.  If we don't, then we can probably remove `react-scripts` as well.

Reply:
Can most likely be removed, the earlier scripts were replaced with `craco`.

Reply:
Done ✅